rename_images.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+files = os.listdir('./data/')
+
+image_info = {}
+for f in files:
+    parts = os.path.splitext(f)[0].split('_')
+    if len(parts) <= 2:
+        continue
+    webcompat_id = parts[0]
+    sequence_no = parts[-2]
+    elem_id = '_'.join(parts[1:-2])
+    browser = parts[-1]
+    if webcompat_id not in image_info.keys():
+        image_info[webcompat_id] = {}
+    image_info[webcompat_id][sequence_no] = elem_id
+    new_name = "%s_%s_%s.png" % (webcompat_id, sequence_no, browser)
+    os.rename("data/" + f, "data/" + new_name)
+
+for webcompat_id, seq_no_and_elem_id in image_info.items():
+    with open("./data/%s.txt" % webcompat_id, "w") as text_file:
+        seq_no_and_elem_id = sorted(seq_no_and_elem_id.items())
+        for value in seq_no_and_elem_id:
+            sequence_no = value[0]
+            elem_id = value[1]
+            text_file.write("%s %s\n" % (sequence_no, elem_id))
```

Comment:
Note that here you're writing text, but in `collect.py` you are writing json objects.
The format of the files should be the same.

Reply:
@marco-c For format to be same I made ```{"id" : "elem_id"}``` per line according to sequence number.

Reply:
It would be better to do `json.dumps({'id': elem_id})`

Reply:
It doesn't matter though, the end result is the same.

Can you run the script on your machine to rename the files? Then zip the resulting `data` directory and upload it on Dropbox or some other service. Before merging this PR, I will have to update `get_dependencies.py` to use the new data.zip file.

Reply:
@marco-c  I will try to upload it today. Got a better Internet connection :smile:

Reply:
@marco-c 
https://www.dropbox.com/s/rsdq581uteu3mtc/data.zip?dl=0 
this is the link with the renamed files.